spec/views/generic_file/new.html.erb_spec.rb

```diff
@@ -16,7 +16,6 @@
       assign(:batch_id, batch_id)
       assign(:user_collections, user_collections)
       allow(controller).to receive(:current_user).and_return(stub_model(User))
-      allow(GenericFilesController).to receive(:upload_complete_path).with(batch_id).and_return("foo")
```

Comment:
It didn't seem to be doing anything in the test since the test passes without it.  The only other test that uses something similar is controllers/generic_files_controller_spec.rb.  So honestly, I don't know why I put it in there in the first place.  It seems like kruft to me, so I removed it.